airbyte-integrations/connectors/source-okta/unit_tests/test_streams.py

```diff
@@ -1,399 +0,0 @@
-#
-# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
-#
-
-import datetime
-import time
-from abc import ABC
-from http import HTTPStatus
-from unittest.mock import MagicMock
-
-import pytest
-import requests
-from airbyte_cdk.models import SyncMode
-from source_okta.source import (
-    CustomRoles,
-    GroupMembers,
-    GroupRoleAssignments,
-    Groups,
-    IncrementalOktaStream,
-    Logs,
-    OktaStream,
-    Permissions,
-    ResourceSets,
-    UserRoleAssignments,
-    Users,
-)
-
-
-@pytest.fixture
-def patch_base_class(mocker):
```

Comment:
Please return the unit tests. They should be aligned with the new approach instead of being deleted.

Reply:
Added unit tests back.